sklearn/ensemble/bagging.py

```diff
@@ -101,7 +101,11 @@ def _parallel_build_estimators(n_estimators, ensemble, X, y, sample_weight,
 
                 curr_sample_weight[not_indices] = 0
 
-            estimator.fit(X[:, features], y, sample_weight=curr_sample_weight)
+            if sample_weight is None:
+                estimator.fit(X[:, features], y)
+            else:
+                estimator.fit(X[:, features], y, sample_weight=curr_sample_weight)
```

Comment:
This change appears to result in failures of `test_bootstrap_samples` and `test_oob_score_regression` in test_bagging.py. (cc: @MechCoder)